src/internal/m365/collection/groups/collection.go

```diff
@@ -86,11 +82,8 @@ func NewCollection(
 	statusUpdater support.StatusUpdater,
 	ctrlOpts control.Options,
 	// doNotMergeItems bool,
-	chanID, chanName string,
 ) Collection {
 	collection := Collection{
-		chanID:   chanID,
-		chanName: chanName,
```

Comment:
Out of curiosity- where will we get channel ID and Name from?

Reply:
From the repoRef path.